### PR TITLE
Add GET /api/public/articles/paginated endpoint

### PR DESCRIPTION
**Pull request by ethanbb**

### CODE REVIEW

File: server/router.js [hunk 1]
```diff
@@ -233,6 +233,38 @@ router.get('/api/group/:groupId/articles/paginated', (req, res) => {
   });
 });
 
+router.get('/api/public/articles/paginated', (req, res) => {
+  const conditions = { pagination: {}, sort: {} };
+
+  // defaults
+  let limit = Number.parseInt(req.query.limit, 10);
```
Comment (ploomiz) [inline server/router.js:240]: Will it still return articles if I don't give it any pagination information in the query?

**Comment by ethanbb** [comment]
Yeah, there are defaults. We can also change the defaults to return all
articles rather than a default # if that makes more sense.

**Review by charblanc — APPROVED** [review]
look at you and pagination ;)

**Review by ploomiz — APPROVED** [review]

**Comment by ethanbb** [comment]
OK merging

**Pull request merged**